reframe/core/modules.py

```diff
@@ -421,7 +421,7 @@ def __init__(self):
             completed = os_ext.run_command('modulecmd -V')
         except OSError as e:
             raise ConfigError(
-                'could not find a sane TMod installation: %s' % e) from e
+                'could not find a sane TMod installation:') from e
```

Comment:
Remove the `:` at the end. The exception chaining mechanism will insert it.

Reply:
Ah yes sorry, you are right! I though I had deleted it